AudioLibrary.WMME/WaveOut.cs

```diff
@@ -0,0 +1,144 @@
+﻿using System;
+using System.Runtime.InteropServices;
+using System.Threading;
+using AudioLibrary.WMME.Native;
+
+namespace AudioLibrary.WMME
+{
+    internal class WaveOut
+    {
+        private IntPtr m_WaveOut;
+        private WaveOutBuffer m_Buffers; // linked list
+        private WaveOutBuffer m_CurrentBuffer;
+        private Thread m_Thread;
+        private BufferFillEventHandler m_FillProc;
+		private bool m_Finished;
+		private byte m_zero;
+
+        private WaveNative.WaveDelegate m_BufferProc = new WaveNative.WaveDelegate(WaveOutBuffer.WaveOutProc);
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Make 'm_BufferProc' 'readonly'. ([csharpsquid:S2933](https://sonarcloud.io/organizations/alex-yevtushenko3820/rules#rule_key=csharpsquid%3AS2933))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=IPP&pullRequest=7&issues=AWfG1-gq8DroMTOoScWn&open=AWfG1-gq8DroMTOoScWn)

[ISSUE_KEY]: AWfG1-gq8DroMTOoScWn